examples/Model.elm

```diff
@@ -169,7 +173,7 @@ update msg ({ clientFormData } as model) =
             )
 
         FetchRecordResponse (Err error) ->
-            model |> updateError error
+            model |> updateError (Kinto.errorToString error)
```

Comment:
`updateError` could also take an error, and then call `Kinto.errorToString` itself, what do you think?